### PR TITLE
LGA-2991 - Disable indexing and whitelisting of Non-Production environments

### PR DESCRIPTION
**Pull request by said-moj**
## What does this pull request do?

Disable indexing and whitelisting of Non-Production environments

## Any other changes that would benefit highlighting?

Pingdom IP addresses also needed to be added to the whitelist. However these frequently change, so an additional script is added to discover these IP addresses.

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

**Review by Pheonnexx — DISMISSED** [review]

File: bin/staging_multideploy.sh [hunk 1]
```diff
@@ -17,5 +17,6 @@ helm upgrade $CLEANED_BRANCH_NAME \
   --set image.repository=$DOCKER_REPOSITORY \
   --set image.tag=$IMAGE_TAG \
   --set dashboard.enabled=false \
+  --set-string pingdomIPs=$PINGDOM_IPS \
```
Comment (digitalali-moj) [inline bin/staging_multideploy.sh:20]: Is this dynamic staging. If so do we need pingdom access? Is it simpler to keep?

**Review by digitalali-moj — APPROVED** [review]
Excellent work! One little question

**Review by TawneeOwl — APPROVED** [review]
LGTM

File: bin/staging_multideploy.sh (hunk 1 above)
Comment (said-moj) [reply bin/staging_multideploy.sh:20]: yes this dynamic staging and no we don't need pingdom access on this environment. However it makes writing the config writing simpler

**Review by Pheonnexx — APPROVED** [review]

**Pull request merged**